### DISCUSSION TITLE
install docker remotely with github action runner

### DISCUSSION THREAD

**Discussion by tdubb123** [community/community#47555] [Actions]
### Select Topic Area

Product Feedback

### Body

how do i use github action runner to install docker remotely on different hosts? Do I need to install the runner on every host ?

**Comment by vitalykarasik** [comment]
> Do I need to install the runner on every host ?

No - you just execute a commands on your target host.
>how do i use github action runner to install docker remotely on different hosts?

Check GHA marketplace. There is general SSH action https://github.com/marketplace/actions/ssh-remote-commands.

**Comment by tdubb123** [comment]
how do I use a github runner to ssh to my onprem servers? Do i need to host
my own runner?

**Reply by vitalykarasik** [reply]
You need either to allow ssh access from Github manager runners, or use self-hosted runner.
